yarn.lock

```diff
@@ -1407,6 +1407,24 @@
   resolved "https://registry.yarnpkg.com/@fluentui-react-native/design-tokens-windows/-/design-tokens-windows-0.13.0.tgz#1d8610634920bd5decaca669cae8183c0713d2ef"
   integrity sha512-o9ir+tpie+Fm9zO+nCbJLBfzVjUhugRWRsPS8n6wBaujC1qyeUYjhoGg6ejXzKc0nVCkIsFmRRRgC0brfrPIzw==
 
+"@fluentui-react-native/framework@0.7.17":
+  version "0.7.17"
+  resolved "https://registry.yarnpkg.com/@fluentui-react-native/framework/-/framework-0.7.17.tgz#b8451227d15640bfcc73b002f7b3f3c19079faf6"
+  integrity sha512-AzI+2c9CbH1+oq1FMYdKzm6GxRup8jj4Gvk+3brECYaKPMHIC+w4iyBPkHdICayc3Iug+N7vlWxduhfaEearaw==
```

Comment:
This getting added means that the Shadow's package.json is referring to an older version of this package. If we bump it to the latest, this should go away.